test/sambah/SAMBAH_Regression.ipynb

```diff
@@ -0,0 +1,363 @@
+    "host_environment = {\n",
+    "    'http://localhost:3000': Environment.LOCAL,\n",
+    "    'https://harmony.uat.earthdata.nasa.gov': Environment.UAT,\n",
+    "    'https://harmony.earthdata.nasa.gov': Environment.PROD,\n",
+    "}\n",
+    "\n",
+    "data_environment = 'PROD' if harmony_host_url == '' else 'UAT'\n",
+    "req_infos = {\n",
+    "    'UAT': {\n",
+    "        # TEMPO NO2 tropospheric, stratospheric, and total columns V03\n",
+    "        'collection': Collection(id='C1262899916-LARC_CLOUD'),\n",
+    "        'temporal': {\n",
+    "            'start': datetime(2024, 3, 28, 15, 34, 0),\n",
+    "            'stop': datetime(2024, 3, 28, 16, 0, 0),\n",
+    "        },\n",
+    "        'spatial': BBox(-170, 33, -10, 38),\n",
+    "        'granule_id': [\n",
+    "            'G1263137623-LARC_CLOUD',\n",
+    "            'G1263137394-LARC_CLOUD',\n",
+    "            'G1263137387-LARC_CLOUD',\n",
+    "            'G1263137388-LARC_CLOUD',\n",
+    "            'G1263137378-LARC_CLOUD',\n",
+    "        ],\n",
+    "        # chosen variables include one variable from each group\n",
+    "        # support/scattering_weights is 3D variable\n",
+    "        'variables': [\n",
+    "            'product/vertical_column_stratosphere',\n",
+    "            'qa_statistics/fit_rms_residual',\n",
+    "            'support_data/scattering_weights',\n",
+    "        ],\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "req_info = req_infos.get(data_environment)\n",
+    "harmony_environment = host_environment.get(harmony_host_url)\n",
+    "\n",
+    "if harmony_environment is not None:\n",
+    "    harmony_client = Client(env=harmony_environment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc7aef85",
+   "metadata": {},
+   "source": [
+    "# Begin regression tests:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9dd58c71-0232-4a20-9dc4-ef6b3544e797",
+   "metadata": {},
+   "source": [
+    "## SAMBAH\n",
+    "\n",
+    "SAMBAH is currently deployed to Sandbox, SIT, UAT and production.\n",
+    "Requests will be made against the TEMPO NO2 L2 V03 collection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef9ec4d2-2228-47d0-a990-d5a4d77b7bc7",
+   "metadata": {},
+   "source": [
+    "### SAMBAH: temporal, variable and bounding box subset request\n",
+    "\n",
+    "This is a request that exercises the full range of SAMBAH options: temporal, bounding box and variable subsetting.\n",
+    "\n",
+    "Requested variable names:\n",
+    "\n",
+    "* `2024-3-28 15:34:00 < time < 2024-3-28 16:00:00`\n",
+    "* `product/vertical_column_total`\n",
+    "* `BBox(-170,30,-10,40)`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6aaf063-1f3c-4d84-b414-ca80e65ef78b",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-05-17T17:55:05.307685Z",
+     "start_time": "2024-05-17T17:53:50.692944Z"
+    }
+   },
+   "outputs": [],
+   "source": [
```

Comment:
A lot of the other test suites wrap their test cells with an `if` condition to guard against when there aren't test collections/granules defined for an environment (mainly production).

Reply:
they now use `if` statements